lms/static/scripts/frontend_apps/config.ts

```diff
@@ -242,9 +242,9 @@ export type EmailPreferences = {
   selectedDays: SelectedDays;
   flashMessage: string | null;
 
-  is_instructor: boolean
-  mention_email_feature_enabled: boolean
-  mention_email_subscribed: boolean
+  is_instructor: boolean;
+  mention_email_feature_enabled: boolean;
+  mention_email_subscribed: boolean;
```

Comment:
This comes from https://github.com/hypothesis/lms/pull/7070, but fixing here for the sake of having a green CI

Reply:
Adding these to mine so it's green as well.